src/scripts/build/getAutoCompleteLocations.js

```diff
@@ -15,6 +15,10 @@ if (!existsSync(dataPath)) {
     const fetchUrl = `https://s3.amazonaws.com/${process.env.S3_BUCKET}/autocomplete/locationAutocompleteData.json`;
     console.debug({ fetchUrl });
     const { body } = await fetch(fetchUrl);
+    if (!body) {
+      throw new Error(`Failed to fetch: ${fetchUrl}`);
+    }
+    // @ts-ignore If body is `null` we are already throwing an error above.
     await finished(Readable.fromWeb(body).pipe(stream));
```

Comment:
```suggestion
    await finished(Readable.fromWeb(body!).pipe(stream));
```

Reply:
I think this is cleaner than ts-ignore and will result in the same thing at runtime I think, *but* I think the real potential problem it's pointing out is that `body` may be non-null but still not a `ReadableStream`. I'm not sure there's much we can do to handle it in this case though, my suggestion should throw a `TypeError` at runtime which is probably the best we can do.

Reply:
Since this is a `.js` file non-null assertion won’t work here. :/

Reply:
Oh shoot right sorry I forgot, yeah that's fine then. Unfortunate to have to turn it off for the whole line that's all. I guess if you care you could explicitly check this and throw.